internal/app/helm_helpers.go

```diff
@@ -55,10 +55,13 @@ func getChartInfo(chartName, chartVersion string) (*chartInfo, error) {
 	}
 
 	constraint, _ := version.NewConstraint(chartVersion)
```

Comment:
good catch, forgot to handle these, looks like we missed the constraint error as well